src/utils/__tests__/clone.test.js

```diff
@@ -29,6 +29,5 @@ describe('clone.js', () => {
     expect(clone(arr)[5]).toEqual(childObj);
     expect(clone(arr)[5] === childObj).toBe(false);
     expect(clone(obj).child).toEqual(childObj);
-    expect(clone(obj).child === childObj).toBe(false);
```

Comment:
clone should be a deep clone, so this test should be readded

Reply:
done